File: c#/bridge/Ejm1/FileLogger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Libs;
namespace Bridge.Ejm1
{
    /// <summary>
    /// Registra un mensaje a un archivo
    /// </summary>
    public class FileLogger : MessageLogger
    {
        public void LogMsg(string msg)
        {
            FileUtil futil = new FileUtil();
            futil.WriteToFile(msg, "log.txt");
        }
    }
}

File: java/bridge/eje1/FileLogger.java
package patrones.bridge.eje1;

/**
 * 
 * Escribe el mensaje en un archivo.
 *
 */
public class FileLogger implements MessageLogger {
	public void logMsg(String msg) {
	    //FileUtil futil = new FileUtil();
	    //futil.writeToFile("log.txt",msg, true, true);
		System.out.println("Mensaje:" +  msg + " grabado en el archivo log.txt");
	  }
}
